js/binance.js

```diff
@@ -4147,21 +4147,27 @@ module.exports = class binance extends Exchange {
             'asset': currency['id'],
             'amount': this.currencyToPrecision (code, amount),
         };
-        request['type'] = this.safeString (params, 'type');
+        const type = this.safeString (params, 'type');
+        if (type !== undefined) {
+            request['type'] = type;
+            params = this.omit (params, 'type');
+        }
         let method = 'sapiPostAssetTransfer';
         if (request['type'] === undefined) {
+            const symbol = this.safeString (params, 'symbol');
+            if (symbol !== undefined) {
+                params = this.omit (params, 'symbol');
+            }
```

Comment:
In this case we must resolve the naming conflict – support both the `symbol` and the `id`, if the exchange-specific param is called `symbol` and the ccxt-param is also called `symbol`. So we must add this:

```suggestion
            let symbol = this.safeString (params, 'symbol');
            if (symbol !== undefined) {
                // support both the marketId and the unified symbol here
                symbol = this.symbol (symbol); 
                params = this.omit (params, 'symbol');
            }
```